website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,76 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to separate values in a [seed](/docs/build/seeds) with a string you provide. The delimiter defaults to comma.
+
```

Comment:
this could be something for the tech review but does it also accept other values such as:

```
seeds:
  <project_name>:
    <seed_directory_name>
      +delimiter: "|"
```

Reply:
That's right!